src/main/java/uk/gov/hmcts/reform/iacaseapi/domain/handlers/presubmit/FeePaymentDisplayProvider.java

```diff
@@ -0,0 +1,32 @@
+package uk.gov.hmcts.reform.iacaseapi.domain.handlers.presubmit;
+
+import static uk.gov.hmcts.reform.iacaseapi.domain.entities.AsylumCaseFieldDefinition.*;
+
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.reform.iacaseapi.domain.entities.AsylumCase;
+
+@Component
+public class FeePaymentDisplayProvider {
+
+    void writeDecisionHearingOptionToCaseData(AsylumCase asylumCase) {
+
+        String hearingFeeOption = asylumCase
+            .read(DECISION_HEARING_FEE_OPTION, String.class).orElse("");
+
+        if (hearingFeeOption.equals(DECISION_WITH_HEARING.value())) {
+
+            asylumCase.write(
+                HEARING_DECISION_SELECTED,
+                "Decision with a hearing. The fee for this type of appeal is £140"
+            );
+
+        } else if (hearingFeeOption.equals(DECISION_WITHOUT_HEARING.value())) {
+
+            asylumCase.write(
+                HEARING_DECISION_SELECTED,
+                "Decision without a hearing. The fee for this type of appeal is £80"
```

Comment:
This fee of £80 can be obtained from payments-api as this is for payment journey.
The fees are only hardcoded once in the ccd-def and its for the start appeal journey.

Reply:
For pay offline, payments api is not called and so the fee is hardcoded here.